test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSFramework.scala

```diff
@@ -61,10 +71,26 @@ final class ScalaJSFramework(
   }
 
   private def frameworkInfoLauncher = {
+    val prefix = optionalExportsNamespacePrefix
     val name = jsonToString(frameworkName.toJSON)
     val code = s"""
-      new org.scalajs.testinterface.internal.InfoSender($name).initAndSend();
+      new ${prefix}org.scalajs.testinterface.internal.InfoSender($name).initAndSend();
     """
     new MemVirtualJSFile(s"testFrameworkInfo.js").withContent(code)
   }
+
+  private[testadapter] def optionalExportsNamespacePrefix: String = {
+    // !!! DUPLICATE code with ScalaJSPlugin.makeExportsNamespaceExpr
```

Comment:
It's the same argument I once gave about I don't know which duplication. The problem is that there is no private common place to put this; and at the same time it shouldn't be public. I don't want to pollute the public API of `tools.javascript` (or `tools`, for that matter) with a relatively ugly implementation detail like this, which is not well specifiable on its own.